apps/language_server/lib/language_server/experimental/provider/handlers/goto_definition.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ElixirLS.LanguageServer.Experimental.Provider.Handlers.GotoDefinition do
+  alias ElixirLS.LanguageServer.Experimental.Protocol.Requests.GotoDefinition
+  alias ElixirLS.LanguageServer.Experimental.Protocol.Responses
+  alias ElixirLS.LanguageServer.Experimental.CodeMod.Location, as: CodeModLocation
+  alias ElixirLS.LanguageServer.Experimental.SourceFile
+  require Logger
+
+  def handle(%GotoDefinition{} = request, _) do
+    source_file = request.source_file
+    pos = request.position
+
+    source_file_string = source_file |> SourceFile.to_string()
+
+    with %ElixirSense.Location{} = location <-
```

Comment:
How can this work? ElixirSense doesn't have these data structures (though it should).

Reply:
actually, It have.
<img width="686" alt="image" src="https://user-images.githubusercontent.com/12830256/220495941-ba51318b-398c-4076-a928-edb136c1a955.png">

Reply:
You're right, I was bamboozled by google, I clicked through to the wrong repo. Mea culpa.
(Google still shows the outdated repo as the #1 search result, despite the fact that it hasn't had a commit in four years)